src/router/index.ts

```diff
@@ -1,6 +1,7 @@
 import { createRouter, createWebHistory } from 'vue-router'
 import TinyMCE from '../views/TinyMCE.vue'
 
```

Comment:
The code patch looks good overall. The commented-out route has been removed, which improves code clarity. However, there's an opportunity to enhance the code's readability and maintainability.```javascript
const routes = [
  {
    path: '/',
    name: 'tiny-mce',
    component: TinyMCE
  },
  // Additional routes can be added here
]

const router = createRouter({
  history: createWebHistory(import.meta.env.BASE_URL),
  routes
})
```